### PR TITLE
removed double charset encoding

### LINKED ISSUE #9419: Export with file option creates invalid commands
When using the export option with the Save as file, option enabled creates some commands that when doing the import of the very same generated file does not import the spanish characters correctly.
This in an example of an export file with the Save as file option enabled:

```
-- phpMyAdmin SQL Dump
-- version 3.3.4-rc1
-- http://www.phpmyadmin.net
--
-- Host: localhost
-- Generation Time: Jun 22, 2010 at 04:45 AM
-- Server version: 5.1.46
-- PHP Version: 5.3.2
SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES latin1 */;
--
-- Database: `carlos_photoalbums`
--
-- --------------------------------------------------------
--
-- Table structure for table `PhotoAlbums`
--

then the table and the data details and at the end:
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;


---------------------------------------------------------------------------------------------------------------------------------------------------------------------

This is the very same export but WITHOUT the Save as file option enabled
-- phpMyAdmin SQL Dump
-- version 3.3.4-rc1
-- http://www.phpmyadmin.net
--
-- Host: localhost
-- Generation Time: Jun 22, 2010 at 04:44 AM
-- Server version: 5.1.46
-- PHP Version: 5.3.2
SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
--
-- Database: `carlos_photoalbums`
--
-- --------------------------------------------------------
--
-- Table structure for table `PhotoAlbums`
--
```

then the table and the data details and at the end there is nothing else.

This behavior is the same using PMA, 332, 333, and the recently released 334.rc1

I use Apache 2.2.11, Php 5.3.2 and MySQL client version: mysqlnd 5.0.7-dev - 091210 - $Revision: 294543 $

If I delete those 6 lines (3 at beginning and the 3 at end) the impor works correctly and the spanish characters are saved fine.

---
- **Original URL**: https://sourceforge.net/p/phpmyadmin/bugs/3049/
- **Original author**: yacar_suscri
- **Found in version**: 3.5.5

### PR DESCRIPTION
**Pull request by manishbisht**
Fixes #9419

Signed-off-by: Manish Bisht <manish.bisht490@gmail.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by manishbisht** [comment]
The only issue with this PR is that it is solving the issue but the query statements are showing empty on the result page.

screenshot from 2017-07-08 23-33-33

**Comment by manishbisht** [comment]
The empty rows issue is always coming when there is the `SET NAMES` statement in the import file. I opened an issue for this https://github.com/phpmyadmin/phpmyadmin/issues/13459

File: libraries/import.lib.php [hunk 1]
```diff
@@ -8,9 +8,11 @@
 use PhpMyAdmin\Encoding;
 use PhpMyAdmin\Message;
 use PhpMyAdmin\Response;
-use PhpMyAdmin\Table;
-use PhpMyAdmin\Util;
-use PhpMyAdmin\Url;
+use PMA\libraries\Table;
+use PMA\libraries\Util;
+use PMA\libraries\URL;
```
Comment (nijel) [inline libraries/import.lib.php]: Seems you're reverting some changes from master here...

Comment (manishbisht) [inline libraries/import.lib.php]: Made mistake during resolving merge conflicts. I have fixed the error and updated the PR.

**Pull request closed without merging**